src/frontend/apps/standalone_site/src/components/Modale/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Button, Layer, LayerProps } from 'grommet';
+import { Nullable } from 'lib-common';
+import { ExitCrossSVG } from 'lib-components';
+import {
+  forwardRef,
+  MutableRefObject,
+  PropsWithChildren,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+const messages = defineMessages({
+  closeButtonTitle: {
+    defaultMessage: 'Close the modale',
+    description: 'Accessibility title for close modale button title.',
+    id: 'components.Modale.closeButtonTitle',
+  },
+});
+
+export interface ModaleApi {
+  open: () => void;
+  close: () => void;
+}
+
+interface ModaleProps extends Omit<LayerProps, 'onClickOutside' | 'onEsc'> {
+  isOpen?: boolean;
+  api?: MutableRefObject<Nullable<ModaleApi>>;
```

Comment:
Why the name api ? I find it a bit confusing.

Reply:
Because it is in fact an API (if you consider the modale as an application). How would you call that ?

Reply:
I understand your logic, but it gives me the feeling that it is to interact with a backend. 
Maybe I would have call that `methods`, I have maybe done too much POO I don't know.

Reply:
Same comment, `api` is confusing